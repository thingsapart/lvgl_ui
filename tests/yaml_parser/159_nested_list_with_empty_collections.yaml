# Tests proper handling of empty collections inside a nested list structure.
- - item_1a
  - {} # empty flow map
  - item_1b
- - item_2a
  - [] # empty flow list
