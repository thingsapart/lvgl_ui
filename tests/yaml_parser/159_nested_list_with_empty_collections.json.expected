[["item_1a", {
		}, "item_1b"], ["item_2a", []]]
